src/main/java/com/mesabrook/ib/util/ModUtils.java

```diff
@@ -4,14 +4,40 @@
 import net.minecraft.util.math.AxisAlignedBB;
 
 import java.net.*;
-import java.util.Scanner;
 
 /**
-** This class provides extra information
+** Immersibrook Utilities Class
 **/
 
 public class ModUtils 
 {
+	/**
+	 * A basic handler that allows us to open the user's default browser and navigate to the provided URL.
+	 *
+	 * @param url URI
+	 */
+	public static void openWebLink(URI url)
+	{
+		try
+		{
+			Class<?> oclass = Class.forName("java.awt.Desktop");
+			Object object = oclass.getMethod("getDesktop", new Class[0]).invoke(null);
+			oclass.getMethod("browse", new Class[]{URI.class}).invoke(object, url);
```

Comment:
This seems over engineered...what's wrong with just doing `Desktop.getDesktop().browse(url);`?

Reply:
Oh, I see. You just moved it up, otherwise this has been in here for some time. Well, I guess nevermind lol